website/.github/workflows/test.yml

```diff
@@ -0,0 +1,97 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
```

Comment:
Does this thing still work correctly?

Reply:
Also, we'll need to change the GitHub pages publishing to start coming from chisel3. I can do this once this switches over.